src/Microsoft.Sbom.Api/Executors/ComponentToPackageInfoConverter.cs

```diff
@@ -52,7 +52,16 @@ async Task ConvertComponentToPackage(ScannedComponent scannedComponent, Channel<
                     try
                     {
                         var sbom = scannedComponent.ToSbomPackage(report);
-                        await output.Writer.WriteAsync(sbom);
+
+                        if (sbom == null)
```

Comment:
Ideally, the adapter should throw and we can act on the failure accordingly. This needs to be fixed in the adapter code, it currently just fails silently.